db/schema.rb

```diff
@@ -0,0 +1,23 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_10_04_202830) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "currencies", force: :cascade do |t|
+    t.string "currencies", null: false
```

Comment:
So here you are banking on that the created_at date tells you which date the exchange rate is from. How will your application work if you need to pull down old rates? What happens if I want you to create a graph, like in the last sub task (which was a hint) where we look at old records from years back?